enterprise/backend/src/metabase_enterprise/advanced_config/settings.clj

```diff
@@ -0,0 +1,2 @@
+(ns metabase-enterprise.advanced-config.settings
+  "Settings related to functionality enabled by the `:advanced-config` token feature.")
```

Comment:
why is this empty? should `subscription-allowed-domains` be in this ns since it is a setting only relevant when `(premium-features/enable-advanced-config?)` is true?

Reply:
oops. I meant to delete this namespace when I moved `subscription-allowed-domains` over

Reply:
fixed